### ISSUE TITLE
Save state corrupted on phone call

### ISSUE THREAD

**Issue by DarthChillash** [rileytestut/Delta#316]
Every time I accept an incomming call while playing Soulsilver Delta overwrites my quicksave with a "white screen" seemingly corrupt state.

Version 1.5.1 from AppStore.

![image](https://github.com/rileytestut/Delta/assets/1872435/521343d9-8b82-4067-b0b6-da18ecc3a261)

**Comment by imticho** [comment]
Does this still happen to you? I tried recreating it on my own and found no success.

**Comment by DarthChillash** [comment]
Yes, every time. Try like this: play a DS game like Soulsilver, get a phone call, there will be a phone call pop-up at the top of the screen, press answer on the incomming call pop-up, then place the phone on your ear, then talk for a few seconds, then remove the phone from ypur ear and hang up on the caller side. Screen goes white and quick save is now corrupted.

If you want I will record it in video.
